docs/src/main/paradox/scala/http/routing-dsl/testkit.md

```diff
@@ -110,6 +110,14 @@ This is what the full working test looks like:
 @@snip [TestKitFragmentSpec.scala](../../../../../test/scala/docs/http/scaladsl/server/TestKitFragmentSpec.scala) { #source-quote }
 
 
+## Increase Timeout
+
+The default timeout when testing your routes using the testkit is 1 second. Sometimes, though, this might not be enough.
+In order to extend this default timeout, to say 5 seconds, just add the following implicit in scope:
```

Comment:
Yes, I think it would be good to add a bit of Scala-specific documentation about the possibility of dilation here, and indeed create an issue to introduce dilation for Java as well.